explorer/src/components/account/AnchorAccountCard.tsx

```diff
@@ -1,63 +1,62 @@
 import React, { useMemo } from "react";
-
 import { Account } from "providers/accounts";
-import { Address } from "components/common/Address";
+import { useCluster } from "providers/cluster";
 import { BorshAccountsCoder } from "@project-serum/anchor";
-import { capitalizeFirstLetter } from "utils/anchor";
+import { IdlTypeDef } from "@project-serum/anchor/dist/cjs/idl";
+import { getProgramName, mapAccountToRows } from "utils/anchor";
 import { ErrorCard } from "components/common/ErrorCard";
-import { PublicKey } from "@solana/web3.js";
-import BN from "bn.js";
-
-import ReactJson from "react-json-view";
-import { useCluster } from "providers/cluster";
 import { useAnchorProgram } from "providers/anchor";
 
 export function AnchorAccountCard({ account }: { account: Account }) {
+  const { lamports } = account;
   const { url } = useCluster();
-  const program = useAnchorProgram(
-    account.details?.owner.toString() ?? "",
+  const anchorProgram = useAnchorProgram(
+    account.details?.owner.toString() || "",
```

Comment:
Interesting; why? Ultimately it won't made a difference since `"" ?? ""` and `"" || ""` will produce the same outcome but I'm curious.

Reply:
Same as above haha, just a manual revert and I guess I'm used to using `||`. Definitely no big reason behind.